### PR TITLE
AS corrections for new API Version

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: otcextensions/osclient/auto_scaling/v1/group.py [hunk 1]
```diff
@@ -337,84 +369,83 @@ def get_parser(self, prog_name):
...
         parser.add_argument(
-            '--availability_zone',
+            '--availability-zone',
             metavar='<availability_zone>',
             action='append',
             help=_('Specifies the availability zones information '
                    '(Repeat multiple times)')
         )
         parser.add_argument(
             '--subnetwork',
```
Comment (gtema) [inline otcextensions/osclient/auto_scaling/v1/group.py]: can you please check this param name? we normally use "subnet" if talk really about subnet. Additionally help text here is still confusing

File: otcextensions/tests/unit/sdk/auto_scaling/v1/test_policy.py [hunk 2]
```diff
@@ -137,7 +137,7 @@ def test_create(self):
         call_args = self.sess.post.call_args_list[0]
 
         expected_json = copy.deepcopy(EXAMPLE)
-        expected_json.pop('scaling_group_id')
+        # expected_json.pop('scaling_group_id')
```
Comment (gtema) [inline otcextensions/tests/unit/sdk/auto_scaling/v1/test_policy.py]: please drop commented line here

File: otcextensions/tests/unit/sdk/auto_scaling/v1/test_policy.py [hunk 3]
```diff
@@ -184,7 +184,7 @@ def test_update(self):
 
         expected_json = copy.deepcopy(EXAMPLE)
         expected_json.pop('scaling_policy_id')
-        expected_json.pop('scaling_group_id')
+        # expected_json.pop('scaling_group_id')
```
Comment (gtema) [inline otcextensions/tests/unit/sdk/auto_scaling/v1/test_policy.py]: and here

**Review by gtema — CHANGES_REQUESTED** [review]
few small fixes and good to go

File: otcextensions/osclient/auto_scaling/v1/group.py (hunk 1 above)
Comment (tischrei) [reply otcextensions/osclient/auto_scaling/v1/group.py]: This is the network ID, not the subnet. Shall I rename it?

Comment (gtema) [reply otcextensions/osclient/auto_scaling/v1/group.py]: yes, but we have already here also network-id, which is mapped to vpc_id - to be fixed also. Same renaming should be done also in UpdateAutoScalingGroup

**Comment by gtema** [comment]
recheck

**Review by gtema — APPROVED** [review]

**Pull request merged**